### PR TITLE
replace access token after revoke

### PR DESCRIPTION
**Pull request by paradox70**
After calling revoke_access_token(), access_token revokes and new access_token must be replace.

### CODE REVIEW

File: telegraph/api.py [hunk 1]
```diff
@@ -109,14 +109,19 @@ def edit_account_info(self, short_name=None, author_name=None,
...
             if the user would like to reset all connected sessions, or
             you have reasons to believe the token was compromised.
             On success, returns dict with new access_token and auth_url fields
         """
 
-        return self._telegraph.method('revokeAccessToken')
+        response = self._telegraph.method('revokeAccessToken')
+        
+        if replace_token:
+            self._telegraph.access_token = response.get('access_token')
```
Comment (python273) [inline telegraph/api.py]: hmm, I think `replace_token` arg is not really needed here

Comment (paradox70) [reply telegraph/api.py]: If you agree with replacing access_token, replacing can be automated.

Comment (python273) [reply telegraph/api.py]: Yeah, I think we can just replace current token, since it gets invalid anyway

File: telegraph/api.py [hunk 2]
```diff
@@ -116,7 +116,11 @@ def revoke_access_token(self):
             On success, returns dict with new access_token and auth_url fields
         """
 
-        return self._telegraph.method('revokeAccessToken')
+        response = self._telegraph.method('revokeAccessToken')
+        
+        self._telegraph.access_token = response.get('access_token')
+
```
Comment (python273) [inline telegraph/api.py]: Can you also remove spaces from blank lines?

**Comment by python273** [comment]
👍

**Comment by paradox70** [comment]
Thanks for this simple and great warper.

**Pull request merged**